### ISSUE TITLE
Message Bubbles - Layout for text messages - Android

### ISSUE THREAD

**Issue by jakewb-b** [vector-im/element-android#4574]
- As a user with message bubbles enabled
- I see text messages within message bubbles, in all rooms and all DMs, with my messages at the right and all other messages at the left.
- So that I have a more familiar interface and can easily differentiate my messages from other peoples

https://www.figma.com/file/zVtAKK0hzKu5JkpbzlE1pG/Message-Bubbles?node-id=1%3A29

NB - to aid in identifying the major tasks, issues have been created for specific items but this should not be taken to mean that is the only functionality to be implemented. The intent is that users with message bubbles enabled will have all the same Element functionality as users without message bubbles enabled.

**Comment by daniellekirkwood** [comment]
![Screenshot 2022-01-13 at 16 58 52](https://user-images.githubusercontent.com/89144281/149385756-b7fe912a-f0c0-4e71-bc96-a4b30dd861a7.png)

@ganfra in testing today we saw that the senders messages are an incorrect width (they come too far left) @amshakal will be able to provide the padding if needed but I think it's in the figma above 👍

**Comment by daniellekirkwood** [comment]
@ganfra the bubbles doesn't render correctly when there's code included... the screenshot here shows the text (that's text) doesn't wrap (and it should) and it's in the wrong font.

![Screenshot_20220113-171145_Element dbg](https://user-images.githubusercontent.com/89144281/149388495-371b6520-4a06-4923-bdec-d29c7a743842.jpg)

**Comment by ganfra** [comment]
Yes code block rendering is kind of broken, not only for bubble. I'll try to fix that

**Comment by daniellekirkwood** [comment]
This issue is complete. Closing.
